src/compat/authcompat.py.in.in

```diff
@@ -296,6 +296,7 @@ class Configuration:
             }
 
             config.write(keys)
+            os.chmod(Path.System('sssd.conf'), mode=0o600)
 
```

Comment:
The change looks safe and simple but the bugzilla talks about permissions on snippet 

    File /etc/sssd/conf.d/authconfig-sssd.conf did not pass access check. Skipping.

Is there some logic that will apply the mode to snippets too?

Reply:
OK I found it in the code :-) Thanks for the patch.

Reply:
Sorry, my python-fu was quite bad five years ago :-)